sql/core/v1.2/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcFilterSuite.scala

```diff
@@ -39,7 +39,7 @@ import org.apache.spark.sql.types._
  * OrcFilterSuite and HiveOrcFilterSuite is logically duplicated to provide the same test coverage.
  * The difference are the packages containing 'Predicate' and 'SearchArgument' classes.
  * - OrcFilterSuite uses 'org.apache.orc.storage.ql.io.sarg' package.
- * - HiveOrcFilterSuite uses 'org.apache.hadoop.hive.ql.io.sarg' package.
+ * - HiveOrcFilterSuite uses $"org".apache.hadoop.hive.ql.io.sarg' package.
```

Comment:
Shall we revert this line?

Reply:
Oops, my search and replace was too aggressive